lib/client/mfa/cli.go

```diff
@@ -185,14 +185,7 @@ func (c *CLIPrompt) Run(ctx context.Context, chal *proto.MFAAuthenticateChalleng
 		fmt.Fprintf(c.writer(), msg, strings.Join(availableMethods, ", "), strings.Join(chosenMethods, " and "))
 	}
 
-	// DELETE IN v18.0 after TOTP session MFA support is removed (codingllama)
-	// Technically we could remove api/mfa.WithPromptChallengeExtensions along
-	// with this, as it's likely its only use, although arguably keeping it could
-	// prove useful.
-	usageSessionMFA := c.cfg.Extensions.GetScope() == mfav1.ChallengeScope_CHALLENGE_SCOPE_USER_SESSION
-	if promptOTP && usageSessionMFA {
-		fmt.Fprint(c.writer(), "\nWARNING: Starting with Teleport 18, OTP will not be accepted for per-session MFA.\n\n")
-	}
+	useSessionMFA := c.cfg.Extensions.GetScope() == mfav1.ChallengeScope_CHALLENGE_SCOPE_USER_SESSION
```

Comment:
Between `usageSessionMFA` and `useSessionMFA` and what actually is behind this check, I'm a bit confused as to what this variable describes. 😅 Did you mean to call it `userSessionMFA`? Maybe something like `isUserSessionMFA` would work too?

Reply:
I replaced this with `isPerSessionMFA` since that's what we call the configuration, but I'm also not against `isUserSessionMFA` if you think that reads more clearly?